bin/gce-delete-instance

```diff
@@ -17,7 +17,6 @@ cmd="gcloud compute instances delete $instname"
 case $key in
   -q|--quite)
   cmd="$cmd -q"
-  shift
```

Comment:
This is an extra shift to parse argument because ``quite`` option should has no argument.